### ISSUE TITLE
bug(md-input): md-hint is partially hidden

### ISSUE THREAD

**Issue by jean-merelis** [angular/material2#863]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
md-hint should be fully displayed

#### What is the current behavior?
md-hint is partially hidden

#### What are the steps to reproduce?
https://material2-app.firebaseapp.com/

![snapshot5](https://cloud.githubusercontent.com/assets/1755960/16861828/c3eea6b4-4a1a-11e6-912e-bfc73946de00.png)

#### Which versions of Angular, Material, OS, browsers are affected?
Angular 2.0.0-rc.4
material alpha.6
Chrome 51

**Comment by jean-merelis** [comment]
A solution is change css from:
```css
  .md-input-wrapper {
    margin: 16px 0;
  }

  .md-hint {
    ...
    bottom: -0.5em;
    ...
  }
```
To:
```css
  .md-input-wrapper {
    margin: 16px 0 22px 0;
  }

  .md-hint {
    ...
    bottom: 0;
    ...
  }
```

**Comment by Dona278** [comment]
Close with #871

**Comment by jelbourn** [comment]
Fixed via #871
